spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -415,6 +421,36 @@ void ClientDLL::ResetAfterframesQueue()
 
 void ClientDLL::OnFrame()
 {
+#if !defined(OE)
+	extern int GetEntityCount();
+	extern CBaseEntity* GetEntity(int);
+
+	if (y_spt_piwsave.GetString()[0] != '\0')
+	{
+		int count = GetEntityCount();
+		for (int i = 0; i < count; ++i)
```

Comment:
There probably is something that can be hooked, but going through the entity list once each frame is not something that causes lag.